ui/raidboss/data/05-shb/raid/e2s.js

```diff
@@ -356,33 +269,15 @@
       condition: function(data, matches) {
         return data.me == matches.target && data.spell[data.me] == 'fire';
       },
-      alertText: {
-        en: 'Spread',
-        de: 'Verteilen',
-        fr: 'Dispersez-vous',
-        ja: '散開',
-        cn: '散开',
-        ko: '산개',
-      },
+      response: Responses.spread('alert'),
     },
     {
       id: 'E2S Shadoweye No Waiting',
       regex: Regexes.headMarker({ id: '00B3' }),
       condition: function(data) {
         return !data.waiting;
       },
-      alertText: function(data, matches) {
-        if (data.me != matches.target) {
-          return {
-            en: 'Look Away from ' + data.ShortName(matches.target),
-            de: 'Von ' + data.ShortName(matches.target) + ' weg schauen',
-            fr: 'Ne regardez pas '+ data.ShortName(matches.target),
-            ja: data.ShortName(matches.target) + 'を見ないで',
-            cn: '背对 ' + data.ShortName(matches.target),
-            ko: '"' + data.ShortName(matches.target) + '" 바라보지 마세요',
-          };
-        }
-      },
+      response: Responses.lookAwayFrom('alarm'),
```

Comment:
/home/travis/build/quisquous/cactbot/ui/raidboss/data/05-shb/raid/e2s.js: E2S Shadoweye No Waiting cannot have both 'response' and 'infoText'

As it says, you have both response and infoText here